packages/babel-parser/test/fixtures/experimental/export-extensions/default-type-without-flow/options.json

```diff
@@ -1,4 +1,5 @@
 {
   "plugins": ["exportDefaultFrom"],
-  "sourceType": "module"
-}
+  "sourceType": "module",
+  "throws": "This experimental syntax requires enabling the parser plugin: 'flow' (1:7)"
```

Comment:
There are three cases we need to make sure we have tests for,

1. `plugins: []`, this should return a syntax error and suggest turning on `exportDefaultFrom`.

1. `plugins: ['exportDefaultFrom']`, this should parse correctly.

1. `plugins: ['flow']`, this should return a syntax error.

Reply:
got it